docs/frontend.md

```diff
@@ -139,7 +139,7 @@ writing these tests in TypeScript, there is documentation in supporting that [he
 The project is now using [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/).
  It originally started with Enzyme, and the decision was made to switch to RTL,
  so there may still be tests that have not been converted.
- (See [ADR](https://github.com/CMSgov/easi-app/blob/master/docs/adr/0028-use-react-testing-library.md)
+ (See [ADR](https://github.com/CMSgov/mint-app/blob/master/docs/adr/0028-use-react-testing-library.md)
```

Comment:
This can probably get updated, as we don't have this specific ADR anymore (maybe just point to the first ADR in the repo that links back to EASi as a base?)